Services/CO.CDP.EntityVerification/Persistence/IPponRepository.cs

```diff
@@ -0,0 +1,12 @@
+namespace CO.CDP.EntityVerification.Persistence;
+
+public interface IPponRepository 
+{
+    public void Save(EntityVerificationContext context, Ppon identifier);
```

Comment:
Now we have an abstraction (IPponRepository) that depeneds on something that the concrete dependency requires. Anything stopping us from passing the context to the DatabasePponRepository?

Reply:
You mean in the constructor for DatabasePponRepository ?

Reply:
Yes, just like we have in other repositories: https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/f92ee6007e738d73d0704a8ae753e3528c324e39/Services/CO.CDP.OrganisationInformation.Persistence/DatabaseTenantRepository.cs#L6